heeji/한-번만-등장한-문자.js

```diff
@@ -0,0 +1,12 @@
+function solution(s) {
+    const result = [];
+    const count = {};
+    for (let i = 0; i < s.length; i++) {
```

Comment:
```suggestion
    for (let i in s) {
```
@heejiyang  삼항연산자도 활용하셔서 잘 푸셨네요!! 이렇게 for in 문을 활용하시면 더 간단히 반복문을 작성하실 수 있습니다 👀